packages/rocketchat-livechat/.app/i18n/ro.i18n.json

```diff
@@ -3,12 +3,12 @@
   "Appearance": "Aspect",
   "Are_you_sure_do_you_want_end_this_chat": "Sigur doriți să încheiați acest chat?",
   "Are_you_sure_do_you_want_end_this_chat_and_switch_department": "Sigur doriți să încheiați această discuție și să schimbați departamentul?",
-  "Cancel": "Anula",
+  "Cancel": "Anulează",
   "Change": "Schimbare",
-  "Chat_ended": "Chat sa încheiat!",
+  "Chat_ended": "Chat-ul sa încheiat!",
```

Comment:
@spirlici Most probably it has to be "Chat-ul s-a încheiat!"

Reply:
Yes, you're right, thanks